src/bootstrap/toolstate.rs

```diff
@@ -325,11 +325,11 @@ fn prepare_toolstate_config(token: &str) {
             Err(_) => false,
         };
         if !success {
-            panic!("git config key={} value={} successful (status: {:?})", key, value, status);
+            panic!("git config key={} value={} failed (status: {:?})", key, value, status);
```

Comment:
This was just a typo, wasn't it?

Reply:
I noticed that, and switched one of them in my PR to "unsuccessful".  My only thought is that maybe it was written like how some people use `expect()` and put the expected result/behavior in the string (like `expect("does not fail")`) instead of explaining the error (`expect("error")`).  I find that quite confusing myself, but I see how reading "expect error" in English is also confusing, since it does the opposite.  I kinda wish a different word than `expect` was chosen.  (Sorry, off-topic ramble.)

Reply:
`unsuccessful` seems fine, too (it was used above so it is more consistent).